### ISSUE TITLE
Update various NIRCam coronagraph reference data

### ISSUE THREAD

**Issue by mperrin** [spacetelescope/webbpsf#115]
<a href="https://github.com/mperrin"><img src="https://avatars0.githubusercontent.com/u/1151745?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [mperrin](https://github.com/mperrin)**
_Thursday Apr 28, 2016 at 05:04 GMT_
_Originally opened as https://github.com/mperrin/webbpsf/issues/115_

----

Per various emails from Stansberry:
- The ND squares are not actually truly neutral, they definitely have some color to them. We could add the appropriate relative spectral response. 
- From https://confluence.stsci.edu/pages/viewpage.action?pageId=52920601: "The wedge element in the SW pupil wheel has chromatic effects which impact PSF shape for wide-bandpass filters, so PSF subtraction will be particularly tricky unless target and PSF reference star are very similar in color.” That’s an effect that is totally neglected in WebbPSF right now.

**Comment by mperrin** [comment]
<a href="https://github.com/mperrin"><img src="https://avatars0.githubusercontent.com/u/1151745?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [mperrin](https://github.com/mperrin)**
_Friday Apr 29, 2016 at 18:13 GMT_

----

Coronagraph wedges info. 

From NIRCam Engineering Memorandum NIR-EM_0424 "Coronagraph Wedge Angle Tolerance Effect on Focal Plane Image Position" (provided by Stansberry):
- SW wedge is BaF_2, angle is 5 degrees
- LW wedge is fused silicon, angle is 0.5 degrees. 
  - pupil stop diameter is 42 mm 
  - note also the pupil wheel mechanical repeatability was estimated to result in up to 2 pixels of PSF shifts in the direction parallel to the axis of the bar occulter. But that was a circa-2007 estimate and is thus not the  most up to date numbers. 

From "NIRCam wedge dispersion trade_Si-BaF2-CaF2-Fused Silica.ppt" for the BaF2 prism

**Comment by JarronL** [comment]
With respect to the wavelength dependence brought on by the Lyot wedges, would it be a process similar to the one we performed for the defocus Zernike component? That is, we would calculate how tip/tilt changes with wavelength using the Zemax data, then modify the OPD map using the tip/tilt offset values?

**Comment by mperrin** [comment]
Yes, that would be the way to do it. Conceptually identical, except for modifying a different coefficient.

**Comment by mperrin** [comment]
@JarronL here is a very old webbpsf issue about improving some reference data, in particular adding the wavelength dependence of the Zernike wedges. I think there is no longer any benefit to keeping this issue open; do you agree? The wavelength dependence of the wedge you implemented a while ago. And the wavelength dependence of the ND filters I don't think we care about enough to add to webbpsf, do we?

**Comment by JarronL** [comment]
This all sounds fine. As long as a LYOT wedge is in place in the `pupil_mask`, then webbpsf should perform the dispersion. So, this should also occur automatically for the ND square observations.
